src/afl-showmap.c

```diff
@@ -1241,11 +1241,15 @@ int main(int argc, char **argv_orig, char **envp) {
     u32 save_be_quiet = be_quiet;
     be_quiet = !debug;
     if (map_size > 4194304) {
-        fsrv->map_size = map_size;
-    }
-    else {
-        fsrv->map_size = 4194304; // dummy temporary value
+
+      fsrv->map_size = map_size;
+
+    } else {
+
+      fsrv->map_size = 4194304;  // dummy temporary value
```

Comment:
Should the default value be in config.h?

Reply:
nah it is more of a dummy value